layouts/_default/article.html

```diff
@@ -4,7 +4,7 @@
     <div class="bg-[#ECF6FF] rounded-b-xl ">
 
         <div class="px-6 pt-4">
-            <h4 class="text-base font-bold text-[#1D65A6]">{{ .Params.category }}</h4>
+            <h4 class="text-base font-bold text-[#1D65A6]">{{ delimit .Params.tags ", " }}</h4>
```

Comment:
Adding `delimit` here ensures that the tags don't appear exactly as they look in the frontmatter.